skimage/data/__init__.py

```diff
@@ -32,18 +37,110 @@
            'horse',
            'hubble_deep_field',
            'immunohistochemistry',
-           'lbp_frontal_face_cascade_filename',
```

Comment:
was this ever meant to be a public image? I kinda removed this from the API. Seemed to be important for tests only.